ale_linters/cpp/clangcheck.vim

```diff
@@ -24,9 +24,9 @@ function! ale_linters#cpp#clangcheck#GetCommand(buffer) abort
     " detected.
     return ale#Escape(ale_linters#cpp#clangcheck#GetExecutable(a:buffer))
     \   . ' -analyze %s'
+    \   . (empty(l:build_dir) ? ' -extra-arg -Xclang -extra-arg -analyzer-output=text' : '')
```

Comment:
Should this be `-Xanalyzer` instead of `-Xclang`?

Reply:
For me, -Xanalyzer yields
> warning: argument unused during compilation: '-Xanalyzer -analyzer-output=text' [-Wunused-command-line-argument]

whereas -Xclang yields no warnings.

Source: https://stackoverflow.com/questions/22371789/how-to-make-the-clang-static-analyzer-output-its-working-from-command-line.

Both seem to report errors on the output though, but -Xclang doesn't throw a warning itself doing so.

Reply:
Okay!